Library/Formula/unibilium.rb

```diff
@@ -0,0 +1,30 @@
+class Unibilium < Formula
+  desc "A very basic terminfo library."
```

Comment:
`* Please remove the indefinite article "A" from the beginning of the description`